src/pages/NewAndEditVRS.page

```diff
@@ -93,36 +93,22 @@
                 <h3 class="slds-section-title slds-text-align--left">{!$Label.labelInformation}</h3>
                 <apex:outputPanel id="main">
                     <apex:repeat value="{!$ObjectType.Volunteer_Recurrence_Schedule__c.FieldSets.NewAndEditVRSFS}" var="f">
-                        <div class="slds-form-element slds-p-top--small slds-size--1-of-2">
-                            <span class="slds-form-element__label">
-                                <apex:outputLabel for="inputField">
-                                    <span class="required slds-required">{!IF(f.DBRequired || f.Required, '*', '')}</span>{!f.Label}
-                                </apex:outputLabel>
-                            </span>
-                            <div class="slds-form-element__control">
-                                <apex:actionRegion >
-                                    <apex:inputField id="inputField" value="{!vrs[f]}" required="{!IF(OR(f.DBRequired, f.Required), 'true', 'false')}"
-                                        styleClass="{!CASE(f.Type,
-                                                        'reference', 'lookupInput',
-                                                        'picklist', 'slds-select',
-                                                        'multipicklist', 'slds-select multipicklistSLDSFixup',
-                                                        'boolean', 'slds-checkbox',
-                                                        'textarea', 'slds-textarea',
-                                                        'date', 'slds-input dateOnlyInput',
-                                                        IF(f == 'Name', '', 'slds-input'))}" >
-                                        <apex:actionSupport immediate="false" event="onchange"
-                                            rerender="jrs" status="loading" rendered="{!IF(f == VJFieldName, 'true', 'false')}"
-                                            action="{!refreshListSOJRS}" />
-                                    </apex:inputField>
-                                </apex:actionRegion>
-                            </div>
-                        </div>
+                        <apex:actionRegion >
+                            <c:UTIL_FormField field="{!f.fieldPath}"
+                                sObj="{!vrs}"
+                                sObjType="Volunteer_Recurrence_Schedule__c"
```

Comment:
I have tested this in a namespaced org and all is well. I believe that when using global variables in VF the namespace gets automagically added. Unfortunately, I can't find any documentation that explicitly states that.